guides/source/testing.md

```diff
@@ -1836,7 +1882,8 @@ test "renders a link to itself" do
 end
 ```
 
-If you'd like to access the same [Capybara-powered Assertions](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Minitest/Assertions)
+If you'd like to access the same [Capybara-powered
+Assertions](https://rubydoc.info/github/teamcapybara/capybara/master/Capybara/Minitest/Assertions)
 that your [Functional and System Testing](#functional-and-system-testing) tests
```

Comment:
This links to a section in "Testing Your Mailers" which isn't expected I think.
I also think these assertions are only available in the SystemTestCases:
```suggestion
that your [System Testing](#system-testing) tests
```